### DISCUSSION TITLE
File type not recognized?

### DISCUSSION THREAD

**Discussion by Sanabria** [konnco/filament-import#74] [Q&A]
Uploading a .csv file works fine in my local machine but thriggers a "The must be a file of type: application/vnd.ms-excel, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, text/csv, text/plain, csv, txt." error right after uploading the .csv in production. I'm using the same .csv file in local and production. Any ideas on what might be happening?

**Comment by frankyso** [comment]
Sometimes its failed to validate the extension, but you can try to remove or add the supported extensions in config file

**Reply by Sanabria** [reply]
@frankyso Thanks for the suggestion but I have already added all possible extensions and still fails somehow. I had to remove manually the acceptedFileTypes method. Perhaps this validation could be a setting in config to avoid this?

**Comment by aliloubm** [comment]
try mime type with python-magic 
i think is "application/octet-stream',"  !!!
